bedrock/externalpages/templates/externalpages/pocket/contact-info.html

```diff
@@ -0,0 +1,54 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% block page_title %}Pocket: Contact Info{% endblock %}
+
+{% block pocket_css %}
+ {{ css_bundle('pocket-contact-info') }}
+{% endblock %}
+
+{% block content %}
+    {% include 'externalpages/pocket/includes/nav.html' %}
+    <section class="mzp-l-content">
```

Comment:
Suggestion: adding `mzp-t-content-xl` here makes the section with a little closer to what's in prod?
```suggestion
    <section class="mzp-l-content mzp-t-content-xl">
```